src/Native/System.Native/pal_mount.h

```diff
@@ -25,31 +25,22 @@ typedef void (*MountPointFound)(const char* name);
 /**
  * Gets the space information for the given mount point and populates the input struct with the data.
  */
-extern "C"
-int32_t GetSpaceInfoForMountPoint(
-    const char*             name, 
-    MountPointInformation*  mpi);
+extern "C" int32_t GetSpaceInfoForMountPoint(const char* name, MountPointInformation* mpi);
 
 /**
- * Gets the format information about the given mount point. 
+ * Gets the format information about the given mount point.
  * We separate format info from space info because format information is given back differently per-platform
- * so keep the space information simple (above) and do all the platform logic here. 
+ * so keep the space information simple (above) and do all the platform logic here.
  * Ubuntu (and most Linux systems) will provide the mount point format as a long int in statfs while
- * OS X and BSD-like systems will provide the information in a char buffer. 
+ * OS X and BSD-like systems will provide the information in a char buffer.
  * Since C# is much better at enum and string handling, pass either the char buffer or the long type
  * back, depending on what the platform gives us, and let C# reason on it in an easy way.
  */
-extern "C"
-int32_t GetFormatInfoForMountPoint(
-    const char* name,
-    char*       formatNameBuffer,
-    int32_t     bufferLength,
-    int64_t*    formatType);
+extern "C" int32_t GetFormatInfoForMountPoint(const char* name, char* formatNameBuffer, int32_t bufferLength, int64_t* formatType);
```

Comment:
GitHub is forcing me to scroll at about the f in formatType. I suggest we use that as the max line length --> 120